src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -110,13 +110,60 @@ public void should_return_a_204_when_claiming_a_task_by_id() {
             .statusCode(HttpStatus.NO_CONTENT.value());
     }
 
+
+    @Test
+    public void should_return_a_204_when_unclaiming_a_task_by_id() {
+
+        String ccdId = ccdIdGenerator.generate();
+
+        List<CamundaTask> tasks = given
+            .iCreateATaskWithCcdId(ccdId)
+            .and()
+            .iRetrieveATaskWithProcessVariableFilter("ccdId", ccdId);
+
+        String taskId = tasks.get(0).getId();
+
+        given
+            .iClaimATaskWithIdAndAuthorization(taskId, authorizationHeadersProvider.getLawFirmAAuthorization());
+
+        Headers headers = authorizationHeadersProvider.getLawFirmBAuthorization();
+
+        Response responseToClaim = given()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .headers(headers)
+            .when()
+            .post("task/{task-id}/unclaim", taskId);
```

Comment:
after unclaim, could we include verification that 
1. the Assignee will bet set to NULL (or absent)
2. the Task State will bet set to "Unassigned"

Reply:
this is done no by @lewis150193